### PR TITLE
Add Weather

### PR DESCRIPTION
**Pull request by SergioMorchon**

### CODE REVIEW

File: types/companion/weather.d.ts [hunk 1]
```diff
@@ -0,0 +1,82 @@
+declare module 'weather' {
+	export const WeatherCondition: {
```
Comment (Hexxeh) [inline types/companion/weather.d.ts:2]: As you've discovered, due to a bug, this is actually missing on Android/sim. Still figuring out what we're going to do here.

File: types/companion/weather.d.ts [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+		readonly temperature: number;
+	}
+
+	export interface Location {
+		readonly name: string;
+		readonly timeOffsetUTC: number;
+		readonly currentWeather: CurrentWeather;
+	}
+
+	export const TemperatureUnit: {
```
Comment (Hexxeh) [inline types/companion/weather.d.ts:60]: Same for this.

Comment (Hexxeh) [reply types/companion/weather.d.ts:60]: These are string valued though, eg `celsius` and `farenheit`, so you could avoid the unknown if you want.

**Review by Hexxeh — APPROVED** [review]

File: types/companion/weather.d.ts (hunk 2 above)
Comment (SergioMorchon) [reply types/companion/weather.d.ts:60]: Nice, thanks for the tips!

**Pull request merged**